src/config/notifier.tsx

```diff
@@ -8,9 +8,10 @@ import { FAILURE_MESSAGES, SUCCESS_MESSAGES } from '@graasp/translations';
 import ToastrWithLink from '../components/common/ToastrWithLink';
 import LIBRARY from '../langs/constants';
 import { SHOW_NOTIFICATIONS } from './env';
-import i18n from './i18n';
+import { i18nConfig } from './i18n';
 import { buildPlayerViewItemRoute } from './paths';
 
+const i18n = i18nConfig();
```

Comment:
Do you still keep the same language since you have a different config?

Reply:
Ah yes that might be an issue ... not sure how I should get the same instance from the `TranslationWrapper` ? D you have an idea ?